### PR TITLE
Add Set-SPOContainer.md, add authors, and remove comments

### PR DESCRIPTION
**Pull request by cindylay**
@abiramisuresh Adding documentation for the new SPO PowerShell command `Set-SPOContainer`

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9

File: sharepoint/sharepoint-ps/sharepoint-online/Set-SPOContainer.md [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+
+
+
+## DESCRIPTION
+
+For any parameters that are passed in, the `Set-SPOContainer` cmdlet sets or updates the setting for the Container collection identified by the parameter Identity.
+
+You must be a SharePoint administrator or Global Administrator to run the cmdlet.
+
+For permissions and the most current information about Windows PowerShell for SharePoint Embedded Containers, see the online documentation at [Intro to SharePoint Embedded Containers Management Shell](https://learn.microsoft.com/powershell/sharepoint/sharepoint-online/introduction-sharepoint-online-management-shell?view=sharepoint-ps).
```
Comment (get-itips) [inline sharepoint/sharepoint-ps/sharepoint-online/Set-SPOContainer.md:39]:
```suggestion
For permissions and the most current information about Windows PowerShell for SharePoint Embedded Containers, see the online documentation at [Intro to SharePoint Embedded Containers Management Shell](/powershell/sharepoint/sharepoint-online/introduction-sharepoint-online-management-shell?view=sharepoint-ps).
```

File: sharepoint/sharepoint-ps/sharepoint-online/Set-SPOContainer.md [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+
+
+## PARAMETERS
+
+
+### -BlockDownloadPolicy
+
+As a SharePoint administrator or global administrator in Microsoft 365, you can block the download of files from SharePoint Embedded Containers. This feature does not need Microsoft Entra Conditional Access policies. This feature can be set for individual Containers but not at the organization level.
+
+Blocking the download of files allows users to remain productive while addressing the risk of accidental data loss. Users have browser-only access with no ability to download, print, or sync files. They also won't be able to access content through apps, including the Microsoft Office desktop apps. When web access is limited, users will see the following message at the top of Containers: "Your organization doesn't allow you to download, print, or sync from this Container. For help contact your IT department." Read the full documentation for advanced capabilities at [Block download policy for SharePoint Containers and OneDrive](https://learn.microsoft.com/sharepoint/block-download-from-Containers).
```
Comment (get-itips) [inline sharepoint/sharepoint-ps/sharepoint-online/Set-SPOContainer.md:85]:
```suggestion
Blocking the download of files allows users to remain productive while addressing the risk of accidental data loss. Users have browser-only access with no ability to download, print, or sync files. They also won't be able to access content through apps, including the Microsoft Office desktop apps. When web access is limited, users will see the following message at the top of Containers: "Your organization doesn't allow you to download, print, or sync from this Container. For help contact your IT department." Read the full documentation for advanced capabilities at [Block download policy for SharePoint Containers and OneDrive](/sharepoint/block-download-from-Containers).
```

File: sharepoint/sharepoint-ps/sharepoint-online/Set-SPOContainer.md [hunk 3]
```diff
@@ -0,0 +1,159 @@
...
+author: abiramisuresh
+ms.author: abisuresh
+ms.reviewer:
+---
+
+# Set-SPOContainer
+
+## SYNOPSIS
+
+Sets or updates one or more properties' values for a Container in SharePoint Embedded.
```
Comment (JohanFreelancer9) [inline sharepoint/sharepoint-ps/sharepoint-online/Set-SPOContainer.md:17]:
```suggestion
Sets or updates one or more property values for a Container in SharePoint Embedded.
```

File: sharepoint/sharepoint-ps/sharepoint-online/Set-SPOContainer.md [hunk 4]
```diff
@@ -0,0 +1,159 @@
...
+```powershell
+Set-SPOContainer [-Identity] 
+ [-BlockDownloadPolicy <Boolean>]
+ [-ConditionalAccessPolicy <SPOConditionalAccessPolicyType>]
+ [-SensitivityLabel <String>] 
+ [-RemoveLabel]
+```
+
+
+
```
Comment (JohanFreelancer9) [inline sharepoint/sharepoint-ps/sharepoint-online/Set-SPOContainer.md:32]:
```suggestion
```

File: sharepoint/sharepoint-ps/sharepoint-online/Set-SPOContainer.md [hunk 5]
```diff
@@ -0,0 +1,159 @@
...
+## DESCRIPTION
+
+For any parameters that are passed in, the `Set-SPOContainer` cmdlet sets or updates the setting for the Container collection identified by the parameter Identity.
+
+You must be a SharePoint administrator or Global Administrator to run the cmdlet.
+
+For permissions and the most current information about Windows PowerShell for SharePoint Embedded Containers, see the online documentation at [Intro to SharePoint Embedded Containers Management Shell](https://learn.microsoft.com/powershell/sharepoint/sharepoint-online/introduction-sharepoint-online-management-shell?view=sharepoint-ps).
+
+
+
```
Comment (JohanFreelancer9) [inline sharepoint/sharepoint-ps/sharepoint-online/Set-SPOContainer.md:42]:
```suggestion
```

File: sharepoint/sharepoint-ps/sharepoint-online/Set-SPOContainer.md [hunk 6]
```diff
@@ -0,0 +1,159 @@
...
+### Example 1
+
+```powershell
+Set-SPOContainer -Identity https://contoso.sharepoint.com/contentstorage/CSP_33a63968-abae-49a3-a255-f83d0ab2260a/ -BlockDownloadPolicy $true
+```
+
+Example 1 sets the Block Download policy on the Container mentioned in the Identity parameter. 
+
+
```
Comment (JohanFreelancer9) [inline sharepoint/sharepoint-ps/sharepoint-online/Set-SPOContainer.md:54]:
```suggestion
```

File: sharepoint/sharepoint-ps/sharepoint-online/Set-SPOContainer.md [hunk 7]
```diff
@@ -0,0 +1,159 @@
...
+### Example 2
+
+```powershell
+Set-SPOContainer -Identity https://contoso.sharepoint.com/contentstorage/CSP_33a63968-abae-49a3-a255-f83d0ab2260a/ -ConditionalAccessPolicy AllowLimitedAccess 
+```
+
+In this example, limited access is provided to content residing in the Container.
+
+
```
Comment (JohanFreelancer9) [inline sharepoint/sharepoint-ps/sharepoint-online/Set-SPOContainer.md:63]:
```suggestion
```

File: sharepoint/sharepoint-ps/sharepoint-online/Set-SPOContainer.md [hunk 8]
```diff
@@ -0,0 +1,159 @@
...
+### Example 4
+```powershell
+Set-SPOContainer -Identity https://contoso.sharepoint.com/contentstorage/CSP_33a63968-abae-49a3-a255-f83d0ab2260a/ -RemoveLabel
+```
+This example sets any previously set sensitivity label on the Container.
+
+
+## PARAMETERS
+
+
```
Comment (JohanFreelancer9) [inline sharepoint/sharepoint-ps/sharepoint-online/Set-SPOContainer.md:80]:
```suggestion
```

File: sharepoint/sharepoint-ps/sharepoint-online/Set-SPOContainer.md [hunk 9]
```diff
@@ -0,0 +1,159 @@
...
+### -BlockDownloadPolicy
+
```
Comment (JohanFreelancer9) [inline sharepoint/sharepoint-ps/sharepoint-online/Set-SPOContainer.md:82]:
```suggestion
```

File: sharepoint/sharepoint-ps/sharepoint-online/Set-SPOContainer.md [hunk 10]
```diff
@@ -0,0 +1,159 @@
...
+As a SharePoint administrator or global administrator in Microsoft 365, you can block the download of files from SharePoint Embedded Containers. This feature does not need Microsoft Entra Conditional Access policies. This feature can be set for individual Containers but not at the organization level.
```
Comment (JohanFreelancer9) [inline sharepoint/sharepoint-ps/sharepoint-online/Set-SPOContainer.md:83]:
```suggestion
As a SharePoint Administrator or Global Administrator in Microsoft 365, you can block the download of files from SharePoint Embedded Containers. This feature does not need Microsoft Entra Conditional Access policies. This feature can be set for individual Containers but not at the organization level.
```

File: sharepoint/sharepoint-ps/sharepoint-online/Set-SPOContainer.md [hunk 11]
```diff
@@ -0,0 +1,159 @@
...
+Applicable: SharePoint Embedded
+Required: False
+Position: Named
+Default value: False
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -ConditionalAccessPolicy
+
```
Comment (JohanFreelancer9) [inline sharepoint/sharepoint-ps/sharepoint-online/Set-SPOContainer.md:102]:
```suggestion
```

File: sharepoint/sharepoint-ps/sharepoint-online/Set-SPOContainer.md [hunk 12]
```diff
@@ -0,0 +1,159 @@
...
+Please read [Control access from unmanaged devices](/sharepoint/control-access-from-unmanaged-devices) documentation here to understand Conditional Access Policy usage in SharePoint Embedded Container.
```
Comment (JohanFreelancer9) [inline sharepoint/sharepoint-ps/sharepoint-online/Set-SPOContainer.md:103]:
```suggestion
Read the [Control access from unmanaged devices](/sharepoint/control-access-from-unmanaged-devices) documentation to understand Conditional Access Policy usage in SharePoint Embedded Container.
```

File: sharepoint/sharepoint-ps/sharepoint-online/Set-SPOContainer.md [hunk 13]
```diff
@@ -0,0 +1,159 @@
...
+
+Possible values:
+- AllowFullAccess: Allows full access from desktop apps, mobile apps, and the web.
+- AllowLimitedAccess: Allows limited, web-only access.
+- BlockAccess: Blocks Access. 
+
```
Comment (JohanFreelancer9) [inline sharepoint/sharepoint-ps/sharepoint-online/Set-SPOContainer.md:109]:
```suggestion
- BlockAccess: Blocks access.
```

File: sharepoint/sharepoint-ps/sharepoint-online/Set-SPOContainer.md [hunk 14]
```diff
@@ -0,0 +1,159 @@
...
+Applicable: SharePoint Embedded
+Required: False
+Position: Named
+Default value: AllowFullAccess
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -SensitivityLabel
+
```
Comment (JohanFreelancer9) [inline sharepoint/sharepoint-ps/sharepoint-online/Set-SPOContainer.md:124]:
```suggestion
```

File: sharepoint/sharepoint-ps/sharepoint-online/Set-SPOContainer.md [hunk 15]
```diff
@@ -0,0 +1,159 @@
...
+Used to specify the unique identifier (GUID) of the SensitivityLabel.
```
Comment (JohanFreelancer9) [inline sharepoint/sharepoint-ps/sharepoint-online/Set-SPOContainer.md:125]:
```suggestion
Specifies the unique identifier (GUID) of the SensitivityLabel.
```

File: sharepoint/sharepoint-ps/sharepoint-online/Set-SPOContainer.md [hunk 16]
```diff
@@ -0,0 +1,159 @@
...
+Aliases:
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+
+
```
Comment (JohanFreelancer9) [inline sharepoint/sharepoint-ps/sharepoint-online/Set-SPOContainer.md:154]:
```suggestion
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @cindylay, kindly commit the suggested changes. Many thanks.

**Review by abiramisuresh — APPROVED** [review]

**Review by msbemba — APPROVED** [review]

**Comment by yogkumgit** [comment]
@tiburd Please help us in resolving this issue. Thanks

**Pull request merged**